index.html

```diff
@@ -0,0 +1,255 @@
+                        <img src="asset/Counter.svg" alt="dot">
+                        <p class="work-para2">2015</p>
+                    </div>
+                    <div class="work-note">
+                        <p>A daily selection of privately 
+                            personalized reads; no accounts or 
+                            sign-ups required.</p>
+                    </div>
+                    <div class="work-tag">
+                        <ul>
+                            <li> <div class="tag">html</div></li>
+                            <li> <div class="tag1">css</div></li>
+                            <li> <div class="tag2">javascript</div></li>
+                        </ul>
+                    </div>
+                    <button type="button" class="see-project">See Project</button>
+                </div>
+            </div>
+
+            <!-- 4TH WORK SECTION -->
+            <div class="work-box1">
+                <div class="work-img">
+                    <img src="asset/Snapshoot-Portfolio2.svg" alt="work_img1">
+                </div>
+                <div class="work-info">
+                    <h3 class="work-title">Multi-Post Stories</h3>
+                    <div class="work-meta">
+                        <p class="work-para">canopy</p>
+                        <img src="asset/Counter.svg" alt="dot">
+                        <p class="work-para1">Back End Dev</p>
+                        <img src="asset/Counter.svg" alt="dot">
+                        <p class="work-para2">2015</p>
+                    </div>
+                    <div class="work-note">
+                        <p>A daily selection of privately 
+                            personalized reads; no accounts or 
+                            sign-ups required.</p>
+                    </div>
+                    <div class="work-tag">
+                        <ul>
+                            <li> <div class="tag">html</div></li>
+                            <li> <div class="tag1">css</div></li>
+                            <li> <div class="tag2">javascript</div></li>
+                        </ul>
+                    </div>
+                    <button type="button" class="see-project">See Project</button>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <!-- MY ABOUT SECTION -->
+<section class="about-section">
+    <div class="about-container">
+        <div class="about-portfolio">
+            <div class="about-box">
+                <h2 class="about-title">About <br> Myself</h2>
+                <p class="about-note">Hello I'm a software developer! I can help you build a product , feature or website Look through
+                    some of my work and experience! If you like what you see and have a project you need to work on,
+                    don't hestiate to contact me.</p>
+                <div class="about-head">
+                    <p>let's connect</p>
```

Comment:
- [ ] Let's update this `p` tag to be an `hx` tag, the purpose of the text included is a call for action more than being a paragraph text.